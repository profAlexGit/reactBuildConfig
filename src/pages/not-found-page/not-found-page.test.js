import React from 'react';
import { configure } from 'enzyme';
import ShallowRenderer from 'react-test-renderer/shallow';
import Adapter from 'enzyme-adapter-react-16';

import NotFoundPage from './not-found-page.jsx';
configure({ adapter: new Adapter() });

describe('<NotFound />', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(<NotFoundPage />);
    expect(tree).toMatchSnapshot();
  });
});
