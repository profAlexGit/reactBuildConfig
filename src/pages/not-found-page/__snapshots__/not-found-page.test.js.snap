// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders correctly 1`] = `
<StatusCode
  code={404}
>
  <main>
    <h1>
      404 page
    </h1>
  </main>
</StatusCode>
`;
